resultstore-search-client/src/components/InvocationTable/index.tsx

```diff
@@ -0,0 +1,120 @@
+*/
+const InvocationTable: React.FC<InvocationTableProps> = ({ invocations }) => {
+    const classes = useStyles();
+    const [page, setPage] = React.useState(0);
+    const [rowsPerPage, setRowsPerPage] = React.useState(10);
+    const rows = invocations.map((invocation) =>
+        parseInvocationTableData(invocation)
+    );
+
+    const handleChangePage = (event: unknown, newPage: number) => {
+        setPage(newPage);
+    };
+
+    const handleChangeRowsPerPage = (
+        event: React.ChangeEvent<HTMLInputElement>
+    ) => {
+        setRowsPerPage(+event.target.value);
+        setPage(0);
+    };
+
+    return (
+        <React.Fragment>
+            <TableContainer className={classes.container}>
+                <Table
+                    stickyHeader
+                    aria-label="sticky table"
+                    className={classes.margin}
+                >
+                    <TableHead>
+                        <TableRow>
+                            {columns.map((column) => (
+                                <TableCell
+                                    key={column.id}
+                                    align={column.align}
+                                    style={{ minWidth: column.minWidth }}
+                                >
+                                    {column.label}
+                                </TableCell>
+                            ))}
+                        </TableRow>
+                    </TableHead>
+                    <TableBody>
+                        {rows
+                            .slice(
+                                page * rowsPerPage,
+                                page * rowsPerPage + rowsPerPage
+                            )
+                            .map((row) => {
+                                return (
+                                    <TableRow
+                                        hover
+                                        role="checkbox"
+                                        tabIndex={-1}
+                                        key={row.name}
+                                    >
+                                        {columns.map((column) => {
+                                            return (
+                                                <TableCell
+                                                    key={column.id}
+                                                    align={column.align}
+                                                >
+                                                    {row[column.id]}
+                                                </TableCell>
+                                            );
+                                        })}
+                                    </TableRow>
+                                );
+                            })}
+                    </TableBody>
+                </Table>
+            </TableContainer>
+            <TablePagination
```

Comment:
Is pagination loading all the data first or loading page by page? If it's loading all the data first, I am going to challenge you to implement pagination on the server side too and make front end load data page by page.

Reason:
A user might search with some very generic keyword, which a lot of data will qualify. If the front end is loading all the data first, then page loading will be super slow and not user friendly.

Reply:
Yea for sure, just want to get a barebones-ish client up and running. Porbably going to setup ci, and test framework next and then iterate and polish the client further.

Reply:
Yeah, this comment is non blocking.